pom.xml

```diff
@@ -30,6 +30,9 @@
       <groupId>org.seleniumhq.selenium</groupId>
       <artifactId>selenium-java</artifactId>
       <version>4.18.1</version>
+      <groupId>org.seleniumhq.selenium</groupId>
+      <artifactId>selenium-java</artifactId>
+      <version>4.18.1</version>
```

Comment:
**Issue**: Duplicate dependency definition for 'selenium-java' which could lead to confusion and potential build issues. <br> **Fix**: Remove the duplicate entries to maintain a clean and error-free pom.xml. <br> **Code Suggestion**: 
 ```
 -      <groupId>org.seleniumhq.selenium</groupId>
 -      <artifactId>selenium-java</artifactId>
 -      <version>4.18.1</version>
 ```